lib/inventory_sync/async/host_result.rb

```diff
@@ -17,7 +17,7 @@ def status_hashes
             {
               host_id: host_id,
               status: InventorySync::InventoryStatus::SYNC,
-              reported_at: DateTime.current,
+              reported_at: Time.current,
```

Comment:
rubocop was annoyed with: `Class: RuboCop::Cop::Style::DateTime`
the diff isn't very critical..
```
[1] pry(main)> Time.current
=> Tue, 04 Aug 2020 08:24:21 UTC +00:00
[2] pry(main)> DateTime.current
=> Tue, 04 Aug 2020 08:24:28 +0000
```